pkg/ios/dsym-utils.go

```diff
@@ -42,7 +42,7 @@ func FindDsymsInPath(path string, ignoreEmptyDsym, ignoreMissingDwarf bool) ([]*
 			tempDir, err = utils.ExtractFile(path, "dsym")
 
 			if err != nil {
-				log.Error("Could not unzip " + fileName + " to a temporary directory, skipping", 1)
+				log.Error("Could not unzip "+fileName+" to a temporary directory, skipping", 1)
```

Comment:
Have we got some local linter settings conflicting with each other? See [previous review comment](https://github.com/bugsnag/bugsnag-cli/pull/79#discussion_r1492609086).

I think you can remove this change from the PR, but it would be good if you can look into checking-in some project linting rules.

Reply:
You're right, for some reason my IDE was using a different linter to the one we have in the `makefile`